### DISCUSSION TITLE
User Interface

### DISCUSSION THREAD

**Discussion by Woutaz** [moneymanagerex/moneymanagerex#4985] [Ideas]
Hello,

To make the UI look better maybe it is worth to add an option of hiding and unhiding the navigation tree, not that it is right now that you must completely hide it and then go to options to get it back.
![image](https://user-images.githubusercontent.com/97412976/187088969-3c4940ca-9a50-4168-b9a2-c401cc80b0d2.png)
![image](https://user-images.githubusercontent.com/97412976/187088978-c207eea9-a4a7-4a1e-8063-d0ea2d0ae0f1.png)

What you think?

**Comment by tactilis** [comment]
@Woutaz  Could you explain how this UI should work please?

What should happen if you click on each of the icons in the collapsed MMEX Navigation tree?

![image](https://user-images.githubusercontent.com/4060092/187092321-27bdf662-1314-4823-8c17-f7cb939c8ca9.png)

It's hard to understand what each of these icons should do, since most of them simply represent the top level nodes in a tree:

![image](https://user-images.githubusercontent.com/4060092/187092409-458075e7-b58e-4f55-956f-2bbc377800c5.png)

You need to expand a branch of the tree to be able to switch between accounts, or to select a different report.

**Reply by Woutaz** [reply]
Hi, 

Now the narrowest option of the navigation tree is as below, if there would be an option of collapsing the navigation then the main screen should automaticly get wider - this would be helpful for the smaller size screen - e.g. laptop.
![image](https://user-images.githubusercontent.com/97412976/187092890-7e1e78f7-0483-4433-ac0d-46ffd47214f2.png)

To achieve wider view now you must to close the navigation tree.

> It's hard to understand what each of these icons should do, since most of them simply represent the top level nodes in a tree:

For example when the tree is collapsed then when you press on one of the options it will work as now, taking you to the main function screen, e.g. "All transactions" would take you down to the transactions screen, the remaining part on which you now press but does not redirect you anywehere  when collapsed should be assigned  by user - maybe by adding an option in General options or just be straight assigned to the top subcategory.

Any thoughts?

**Comment by tactilis** [comment]
> the remaining part on which you now press but does not redirect you anywehere when collapsed should be assigned by user - maybe by adding an option in General options or just be straight assigned to the top subcategory.

That's all pretty complicated given that you'd have to expand the tree to see your accounts just to change to one of them.

It would be much simpler to provide a faster way to show/hide the Navigation pane, for example by adding a shortcut key.

![image](https://user-images.githubusercontent.com/4060092/187094372-6d28d0b5-bac6-4e4f-a3b3-890cc46f40db.png)
